inc/impact.class.php

```diff
@@ -127,16 +135,488 @@ public static function displayTabContentForItem(
+         echo '</tr>';
+         echo '</thead>';
+
+         foreach ($data as $itemtype => $items) {
+            echo '<tbody>';
+
+            // Subheader
+            echo '<tr class="tab_bg_1">';
+            echo '<td class="left subheader impact-left" width="20%">';
+            $total = count($items);
+            echo '<a>' . _n($itemtype, $itemtype, $total) . '</a>' . ' (' . $total . ')';
+            echo '</td>';
+            echo '<td class="left subheader" width="70%"></td>';
+            echo '<td class="subheader" width="10%">';
+            echo '<i class="fas fa-2x fa-caret-down impact-toggle-subitems impact-pointer"></i>';
+            echo '</td>';
+            echo '</tr>';
+
+            foreach ($items as $itemtype_item) {
+               // Content: one row per item
+               echo '<tr class=tab_bg_1><div></div>';
+               echo '<td class="impact-left"><div>' . $itemtype_item['stored']->fields['name']  . '</div></td>';
+               echo '<td><div>';
+
+               $path = [];
+               foreach ($itemtype_item['node']['path'] as $node) {
+                  if ($node['id'] == $start_node_id) {
+                     $path[] = '<b>' . $node['name'] . '</b>';
+                  } else {
+                     $path[] = $node['name'];
+                  }
+               }
+               $separator = '<i class="fas fa-angle-right"></i>';
+               echo implode(" $separator ", $path);
+
+               echo '</div></td>';
+               echo '<td class="center"></td>';
+               echo '</tr>';
+            }
+
+            echo '</tbody>';
+         }
+
+         echo '</table>';
+      }
+
+      echo '</div>';
+
+      $can_update = Session::haveRight(
+         $impact_item->fields['itemtype']::$rightname,
+         UPDATE
+      );
+
+      // Toolbar
+      echo '<div class="impact-list-toolbar">';
+      echo '<a target="_blank" href="../front/impactcsv.php?itemtype=' . $impact_item->fields['itemtype'] . '&items_id=' . $impact_item->fields['items_id'] .'">';
+      echo '<i class="fas fa-2x fa-download impact-pointer"></i>';
+      echo '</a>';
+      if ($can_update) {
+         echo '<i id="impact-list-settings" class="fas fa-2x fa-cog impact-pointer"></i>';
+      }
+      echo '</div>';
+
+      // Settings dialog
+      if ($can_update) {
+         echo '<div id="list_depth_dialog" class="impact-dialog" title=' . __("Settings") . '>';
+         echo '<form action="../front/impactitem.form.php" method="POST">';
+         echo '<table class="tab_cadre_fixe">';
+         echo '<tr>';
+         echo '<td><label>' . __("Max depth") . '</label></td>';
+         echo '<td>' . Html::input("max_depth", [
+            'value' => $impact_item->fields['max_depth'] >= self::MAX_DEPTH ? '' : $impact_item->fields['max_depth'],
+         ]) . '</td>';
+         echo '</tr>';
+         echo '<tr>';
+         echo '<td><label>' . __("No limit") . '</label></td>';
```

Comment:
Lables tags must have a `for` attribute corresponding to the `id` of the form element

Reply:
Fixed in https://github.com/glpi-project/glpi/pull/6441/commits/8c9164fb530c4497f9756205d9c687e64a1eb3da.